packages/react/src/helpers/validateShippingAddressZipCode.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  type AddressBase,
+  type CountryAddressSchema,
+} from '@farfetch/blackout-client';
+
+/**
+ * Zip Code Validation Errors
+ * * ER01 - The zip code validation is ignored because is not included in the countries array to be validated
+ * * ER02 - Zip code schema not found
+ * * ER03 - The zip code inserted is invalid
+ */
+export enum ZipCodeValidationError {
+  ZipCodeValidationCountryIgnored = 'ER01',
+  ZipCodeSchemaNotFoundError = 'ER02',
+  ZipCodeInvalid = 'ER03',
+}
+
+/**
+ * Validates zip code for an array of iso-codes and returns true if the address' country code is not in the array isoCodesToValidate,
+ * or if the passed address' zip code is valid, false otherwise.
+ *
+ * @param address - address
+ * @param addressSchema - address schema to test.
+ * @param isoCodesToValidate - iso codes to be validated.
+ *
+ * @returns - boolean which is the result of the validation.
+ * */
+export default function validateShippingAddressZipCode(
+  address: AddressBase,
+  addressSchema: CountryAddressSchema[],
```

Comment:
Since this is an array, this should be called addressSchemas. Please, change all references to the plural (don't forget the jsdoc as well)

Reply:
done, pls double check